### PR TITLE
add tailwind

### PR DESCRIPTION
**Pull request by marcoswaldo**
# Description
 - add tailwind gem to the project

### CODE REVIEW

**Review by jairolan — DISMISSED** [review]

**Review by JoseRicardoJimenez — DISMISSED** [review]
ok (Y)

File: local_sales/app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag "application", "data-turbo-track": "reload" %>
```
Comment (YilKanda) [inline local_sales/app/views/layouts/application.html.erb:11]: I remember that TurboLinks is also called here to verify it.

```suggestion
    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

**Review by YilKanda — COMMENTED** [review]
I only added one comment, but overall, it looks great.

File: local_sales/app/views/layouts/application.html.erb (hunk 1 above)
Comment (juliocabrera820) [reply local_sales/app/views/layouts/application.html.erb:11]: you're right

File: local_sales/app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (juliocabrera820) [inline local_sales/app/views/layouts/application.html.erb:9]: You can remove this helper

**Review by juliocabrera820 — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by oramasangel — APPROVED** [review]
great

**Review by JoseRicardoJimenez — APPROVED** [review]

**Pull request merged**